system-tests/tests/src/testFixtures/java/org/eclipse/dataspaceconnector/system/tests/utils/GatlingUtils.java

```diff
@@ -40,8 +38,8 @@ public static void runGatling(Class<? extends Simulation> simulation, String des
 
         var statusCode = Gatling.fromMap(props.build());
 
-        assertThat(statusCode)
-                .withFailMessage("Gatling Simulation failed")
-                .isEqualTo(0);
+        if (statusCode != 0) {
+            throw new AssertionError("Gatling Simulation failed");
+        }
```

Comment:
nit: just want to understand, why this change ?

Reply:
Not to take a dependency to assertj in an extensible module just for this.

Reply:
👍 okay